src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.SslProtocols.cs

```diff
@@ -264,9 +257,25 @@ public void DisabledProtocols_SetSslProtocols_ThrowsException(SslProtocols disab
                 var options = new LoopbackServer.Options { UseSsl = true, SslProtocols = acceptedProtocol };
                 await LoopbackServer.CreateServerAsync(async (server, url) =>
                 {
-                    await TestHelper.WhenAllCompletedOrAnyFailed(
-                        Assert.ThrowsAsync(exceptedServerException, () => server.AcceptConnectionSendResponseAndCloseAsync()),
+                    bool gotExpectedException = false;
+                    Task serverTask = server.AcceptConnectionSendResponseAndCloseAsync();
+                    try
+                    {
+                        await TestHelper.WhenAllCompletedOrAnyFailed(serverTask, 
```

Comment:
Nit: the indentation here is strange. At a minimum the next line should be indented, as it's part of this statement.